fastlane/Fastfile

```diff
@@ -0,0 +1,9 @@
+default_platform(:mac)
+
+platform :mac do
+
+  desc "Sets podspec version to the current tag version."
+  lane :set_podspec_version do |args|
+    version_bump_podspec(version_number: args[:version])
```

Comment:
I think if we really need a whole big and heavy Fastlane just for changing the version number in the pod spec. We could create a simple script for that, couldn't we?

Reply:
Yes. We can replace Fastlane doing only this with the script. Consider doing this in the next version of if you already have prepared solution apply it to this version

Reply:
Ok, so I'll create an issue to v1.1 for that

Reply:
Ok. It works for me